Dockerfile

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# syntax=docker/dockerfile:1
+
+######################### BUILD STAGE 1 ############################
+# This stage builds the HarbourBridge binary using its source code.
+
+# Add the official Docker golang image as the base image for the build stage
+FROM golang:latest AS build
+
+# Set the default work directory as “/hbBin” in the container.
+WORKDIR /hbBin
```

Comment:
nit: should we rename this to `harbourbridge` ? keeps it consistent with what it says when we clone it.

Reply:
The binary file name is also `harbourbridge`. The path to the binary will become `/harbourbridge/harbourbridge` if we make this change. I think the current name is more descriptive, it tells that this is the binary directory for  `harbourbridge` and the path to the binary is `/hbBin/harbourbridge`.

Reply:
maybe `harbourbridge_bin` or something which follows `my_dir_type`? I haven't seen directories being named using `camelCase` in linux..

Reply:
Done. `hbBin` > `harbourbridge_bin`.